rlclientlib/serialization/fb_serializer.h

```diff
@@ -242,6 +247,56 @@ namespace reinforcement_learning { namespace logger {
     buffer_t& _buffer;
   };
 
+  template <>
+  struct fb_event_serializer<generic_event> {
+    using fb_event_t = v2::Event;
+    using offset_vector_t = typename std::vector<flatbuffers::Offset<fb_event_t>>;
+    using batch_builder_t = v2::EventBatchBuilder;
+
+    static size_t size_estimate(const generic_event& evt) {
+      return evt.get_payload().size();
+    }
+
+    static int to_fb_payload_type(payload_type type, v2::PayloadType& result, api_status* status) {
```

Comment:
How is this not a numeric cast?

Reply:
even more, it is probably better just to get rid of intermediate enum. will do that